From 2fc1fa6ae451409faf97b4c7c36ecf8abe1f7ce5 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
Date: Sun, 25 Nov 2018 10:28:49 +0900
Subject: [PATCH 2/2] docs: add link to the latest documentation

SPP documentation hosted on dpdk.org is the latest tagged version and
not including the latest topics of developing version. This update is
to add link to the developing version of documentation[1].

[1] https://spp.readthedocs.io/en/latest/

Signed-off-by: Yasufumi Ogawa <ogawa.yasufumi@lab.ntt.co.jp>
---
 docs/guides/index.rst | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/docs/guides/index.rst b/docs/guides/index.rst
index 76c8c79..a01c959 100644
--- a/docs/guides/index.rst
+++ b/docs/guides/index.rst
@@ -13,3 +13,10 @@ SPP documentation
    tools/index
    spp_vf/index
    spp-ctl/index
+
+This documentation is the latest tagged version, but some of the latest
+developing features might be not included.
+All of features not included in this documentation is described in the
+developing version of
+`SPP documentation
+<https://spp.readthedocs.io/en/latest/>`_.
-- 
2.7.4

